index.src.html

```diff
@@ -109,8 +109,9 @@
 :: Refers generically to an instance of AudioDecoder, AudioEncoder,
     VideoDecoder, or VideoEncoder.
 
-: Key Frame
-:: An encoded frame that does not depend on any other frames for decoding.
+: <dfn lt="Key Chunk|Key Frame">Key Chunk</dfn>
+:: An encoded chunk that does not depend on any other frames for decoding. Also
+    commonly referred to as a "key frame".
```

Comment:
We can update this later when we have more definition of the bytestream maybe.

Reply:
Ack, could be good. We should have a first PR out for that soon to see about connecting these dots.